src/integration-test/java/com/commercetools/project/sync/CategorySyncWithReferenceResolutionIT.java

```diff
@@ -39,15 +40,16 @@
 
 public class CategorySyncWithReferenceResolutionIT {
 
-  private static final TestLogger syncerTestLogger = TestLoggerFactory.getTestLogger(Syncer.class);
+  private static final TestLogger categorySyncerTestLogger =
```

Comment:
No need for such a long name I guess. Each class has always a single static logger defined as a class variable and thus can be always defined like that:

`private static final Logger logger = Logger.getLogger(MyClass.class);`

Reply:
https://github.com/commercetools/commercetools-project-sync/pull/264/commits/0f661b72bedf16d82dfb9bc4332fe168ee5ea20f